static/searchPage.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <title>Ogene - Group Search</title>
-    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDDhfcuk15Apx3i72mFSilulsPtJReGhcY"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=APIkey"></script>
```

Comment:
This will send the api key as "APIkey".

Reply:
I'm trying to figure out the best way to load the API key from .env to searchPage.html

Reply:
Hmm... it *could* be done if the searchPage.html file was turned into a template file, and then the API key was templated in, whenever the HTML file was served.

However, this would add a bit of extra complexity to the server, and it might not be worth doing for the MVP. Since the map's API key is visible to any front-end user, it might not be worth the hassle of protecting it through obscurity (since we will have security rules in place anyway).

Reply:
Yeah, I'll look into this in my next PR